src/screens/ProfileScreen/index.js

```diff
@@ -5,24 +5,46 @@ import {
   Text, 
   View,
   Dimensions,
+  TouchableHighlight,
   Image,
-  Picker,
  } from 'react-native';
 
-import profile from '../../../wireframes/assets/Setting_screen/profile.png';
+import profile from '@wireframes/assets/Setting_screen/profile.png';
 import { Entypo } from 'react-native-vector-icons';
 import { Feather } from 'react-native-vector-icons';
 import { MaterialIcons} from 'react-native-vector-icons';
 
+const COLORS = {
+  grey: 'rgb(179, 182, 183)',
+}
+
+export default colors;
```

Comment:
Remove this line. You don't need to export the colors. You can simply access them as `COLORS.grey`

Reply:
yes, I realized that and have edited the same in "update index.js"